apps/storefront/src/pages/UserManagement/table/B3Table.tsx

```diff
@@ -47,210 +30,28 @@ export interface Pagination {
-  lastItemBorderBottom: string;
-}
-
-const MOUSE_POINTER_STYLE = {
-  cursor: 'pointer',
-};
-
-function Row<Row>({
-  columnItems,
-  node,
-  index,
-  showCheckbox,
-  selectCheckbox = [],
-  selectedSymbol,
-  disableCheckbox,
-  showBorder,
-  handleSelectOneItem,
-  clickableRowStyles,
-  lastItemBorderBottom,
-  tableKey,
-  onClickRow,
-  hover,
-  CollapseComponent,
-  applyAllDisableCheckbox,
-}: RowProps<Row>) {
-  const { isCollapse = false, disableCurrentCheckbox } = node;
-
-  const [open, setOpen] = useState<boolean>(isCollapse || false);
-
-  return (
-    <>
-      <TableRow
-        // @ts-expect-error typed previously as an any
-        key={`${node[tableKey || 'id'] + index}`}
-        hover={hover}
-        onClick={() => onClickRow?.(node, index)}
-        sx={clickableRowStyles}
-        data-testid="tableBody-Row"
-      >
-        {showCheckbox && selectedSymbol && (
-          <TableCell
-            key={`showItemCheckbox-${node.id}`}
-            sx={{
-              borderBottom: showBorder ? '1px solid rgba(224, 224, 224, 1)' : lastItemBorderBottom,
-            }}
-          >
-            <Checkbox
-              // @ts-expect-error typed previously as an any
-              checked={selectCheckbox.includes(node[selectedSymbol])}
-              onChange={() => {
-                // @ts-expect-error typed previously as an any
-                if (handleSelectOneItem) handleSelectOneItem(node[selectedSymbol]);
-              }}
-              disabled={applyAllDisableCheckbox ? disableCheckbox : disableCurrentCheckbox}
-            />
-          </TableCell>
-        )}
-
-        {CollapseComponent && (
-          <TableCell>
-            <IconButton aria-label="expand row" size="small" onClick={() => setOpen(!open)}>
-              {open ? <KeyboardArrowDownIcon /> : <KeyboardArrowRightIcon />}
-            </IconButton>
-          </TableCell>
-        )}
-
-        {columnItems.map((column) => (
-          <TableCell
-            key={column.title}
-            sx={{
-              ...column?.style,
-              borderBottom: showBorder ? '1px solid rgba(224, 224, 224, 1)' : lastItemBorderBottom,
-            }}
-            data-testid={column?.key ? `tableBody-${column?.key}` : ''}
-          >
-            {/* @ts-expect-error typed previously as an any */}
-            {column.render ? column.render(node, index) : node[column.key]}
-          </TableCell>
-        ))}
-      </TableRow>
-      {CollapseComponent && (
-        <TableRow>
-          <TableCell style={{ padding: 0 }} colSpan={24}>
-            <Collapse in={open} timeout="auto" unmountOnExit>
-              <CollapseComponent row={node} />
-            </Collapse>
-          </TableCell>
-        </TableRow>
-      )}
-    </>
-  );
+  itemXs: number;
+  onPaginationChange: (pagination: Pagination) => void;
+  pagination: Pagination;
+  rowsPerPageOptions: number[];
+  renderItem: (row: Row) => ReactElement;
```

Comment:
Might want to take the opportunity to rename this to something that implies it is the mobile view?

`renderMobileItem`

Reply:
funnily enough, unlikely most pages, this one uses `renderItem` for both desktop and mobile

there are no actual table elements left in this table component anymore 😆 

I expect `renderItem` to dissolve away with future refactoring efforts